pkg/recipes/driver/bicep.go

```diff
@@ -403,3 +434,28 @@ func (d *bicepDriver) getGCOutputResources(current []string, previous []string)
 
 	return diff, nil
 }
+
+func (d *bicepDriver) FindSecretIDs(ctx context.Context, envConfig recipes.Configuration, definition recipes.EnvironmentDefinition) (secretStoreIDResourceKeys map[string][]string, err error) {
+	secretStoreIDResourceKeys = make(map[string][]string)
+	if envConfig.RecipeConfig.Bicep.Authentication != nil {
```

Comment:
might want to consider nil-checking this in case Authentication, Bicep, or RecipeConfig are nil